docs/docusaurus/static/docs/_static_components/_gx_cloud_advert.md

```diff
@@ -0,0 +1 @@
+[Check out GX Cloud](https://greatexpectations.io/cloud), our SaaS platform—it's now in public preview! [Sign up here](https://greatexpectations.io/cloud) and you could be validating your data in minutes. We also offer regular GX Cloud workshops: [click here to get more information and register](https://pages.greatexpectations.io/gx-cloud-workshops).
```

Comment:
Let's remember to keep an eye on this depending on when we move to GA, which could happen before or after 1.0 is launched. We'll want to update the wording here when it happens.